### ISSUE TITLE
Corrupt Player Data Crafting 2 Haste Modules

### ISSUE THREAD

**Issue by Jo-Mamma75** [Poke1650/XP-Plus#7]
Minecraft Version:  1.10.2
Forge Version:  2239
Other Mods installed (leave blank if none):  FoolCraft 1.2.2
Crashlogs (via Pastebin, Hastebin etc.):  N/A
Steps to reproduce:  Craft 2 Haste Modules
Screenshots (if relevant):  N/A
Other mods/Modpack:  See above
Multiplayer or Singleplayer?:  Multiplayer Server

This issue was posted on our FoolCraft Github ([Issue#220](https://github.com/scalda/FoolCraft/issues/220)) but thought it would be better answered by you.
Basically the guy was saying that he had crafted 2 Haste Modules and he crashed out (no, he didn't provide a crashlog) and his player data "had to be reset" (because whoever is running the server didn't know how or was too lazy to edit it).
Let me know if it's actually an issue.  I haven't taken the time to reproduce the issue on any of our servers.
Thanks in advance.

Jo_Mamma75
Iskall85 Community Administrator
FoolCraft Dev Team

**Comment by antoinegag** [comment]
Hi,
looks like an issue that happened in version 1.01 that was fixed in version 1.02 (See issue #6 ).
I think Foolcraft 1.2.2 is currently using v1.01 and if that's the case updating to v1.02 should fix it.

If that's not it I'll further investigate,

thanks

**Comment by Jo-Mamma75** [comment]
I'll let you know once we finalize the 1.3 update of the pack.
Thanks for letting me know.  I appreciate it.

**Comment by antoinegag** [comment]
I assume this is fixed now
